package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@streammedev/parrot-server",
+  "version": "1.0.0",
+  "description": "Simple http server which returns a body with information about the request such as method, pathname, query parameters, and the request body. It is useful for testing.",
+  "main": "index.js",
+  "keywords": [
+    "test"
+  ],
+  "bin": {
+    "parrot-server": "./bin/parrot-server"
+  },
+  "author": {
+    "name": "StreamMe",
+    "url": "http://github.com/streammedev"
+  },
+  "license": "UNLICENSED",
+  "repository": {
+    "type": "git",
+    "url": "ssh://git@git.vubeology.com/npm/simple-test-server.git"
+  },
+  "publishConfig": {
+    "registry": "http://npm-onsite-internal.stream.me:8080"
+  },
+  "scripts": {
+    "test": "npm run babel && happiness && mocha --compilers js:babel-register --grep 'BROWSER: ' --invert",
+    "prepublish": "npm test",
+    "postpublish": "git push $$ git push --tags",
+    "test-browser": "mochify",
+    "babel": "babel src -d lib",
+    "dev": "babel src -d lib --watch",
+    "api-docs": "rm API.md && jsdoc2md src/index.js >> API.md"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.16.0",
+    "babel-preset-es2015": "^6.16.0",
+    "babel-register": "^6.18.0",
+    "commander": "^2.9.0",
+    "happiness": "^7.1.2",
+    "jsdoc-to-markdown": "^2.0.1",
+    "mocha": "^3.1.2",
+    "mochify": "^2.18.1",
+    "request": "^2.79.0"
+  },
+  "dependencies": {
+    "assert": "^1.4.1",
```

Comment:
What do you use `assert` for?  Do you mean to just use the node core `assert` package?